packages/turbo/ios/vendor/scripts/install-turbo-ios.sh

```diff
@@ -0,0 +1,13 @@
+TURBO_IOS_REPO_PATH="https://github.com/hotwired/turbo-ios.git"
+
+# First argument is the version tag
+if [ -z "$1" ]
+  then
+    echo "No version tag supplied"
+    exit 1
+fi
+
+# Shallow clone the turbo-ios repo
+cd ./ios/vendor
+rm -rf turbo-ios
+git clone --branch $1 --depth 1 $TURBO_IOS_REPO_PATH
```

Comment:
I'm thinking that we should probably include `.framework` file and not clone the repo when building the app, just use `turbo-ios` as a "local lib".